### ISSUE TITLE
For same url, Download speed ofis much much lower than hitomi_downloader_GUI

### ISSUE THREAD

**Issue by lukemin** [HDoujinDownloader/HDoujinDownloader#114]

**Comment by HDoujinDownloader** [comment]
If you go to **_Tools > Site preferences > Hitomi.la_** from the menu at the top of the program window, is "Download WebP images" checked? If not, make sure it is. Otherwise, it will attempt (and fail) to download JPEG and PNG images instead. This setting is enabled by default for newer users, but older users may have to enable it manually.

**Comment by lukemin** [comment]
"Download WebP images" checked，I tooked the screen shots at the the same time.

**Comment by HDoujinDownloader** [comment]
I'll look into it, then. 

Out of curiosity, do you find them to be this slow consistently? Do they continue to be slow if you use default settings (i.e. a fresh copy of HDoujin Downloader)?

**Comment by lukemin** [comment]
yes, I installed the two apps on my windows VPS freshly. Firstly I installed hdoujin downloader, I found that the download speed of hdoujin downloader was too slow.  then I installed hitomi downloader and  took the screenshots.
